srcpkgs/cyme/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'cyme'
+pkgname=cyme
+version=2.0.0
+revision=1
+build_style=cargo
+short_desc="List system USB buses and devices"
+maintainer="klardotsh <josh@klar.sh>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/tuna-f1sh/cyme"
+changelog="https://raw.githubusercontent.com/tuna-f1sh/cyme/main/CHANGELOG.md"
+distfiles="https://github.com/tuna-f1sh/cyme/archive/refs/tags/v${version}.tar.gz"
+checksum=2b97ac1560ec96d28be21c1dfda62802c0149fc445fa64bd9df4da9bee8c9f16
+
+post_install() {
+	vcompletion doc/cyme.bash bash
+	vcompletion doc/_cyme bash
+	vcompletion doc/cyme.fish fish
+
+	vman doc/cyme.1
+
+	vmkdir usr/share/examples/${pkgname}/
+	vcopy doc/cyme_example_config.json usr/share/examples/${pkgname}/config.json
```

Comment:
```suggestion
	vsconf doc/cyme_example_config.json config.json
```

Reply:
That's a crafty alias I didn't know we had, thanks!